bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/transceiver/EnOceanTransceiver.java

```diff
@@ -238,14 +238,15 @@ public void shutDown() {
         logger.debug("shutting down transceiver");
         logger.debug("Interrupt rx Thread");
 
-        Future<?> localTimeOut = timeOut;
-        if (localTimeOut != null) {
-            localTimeOut.cancel(true);
+        Future<?> timeOut = this.timeOutTask;
+        if (timeOut != null) {
+            timeOut.cancel(true);
```

Comment:
Just one minor comment. For consistency, you could name your variable `timeOutTask`

Reply:
Of course, I forgot that. Fixed.